job_template.sh

```diff
@@ -0,0 +1,11 @@
+#SBATCH --account=def-sabeda
+#SBATCH --time=0-08:00        # time (DD-HH:MM) |TODO to change
+#SBATCH --nodes=1
+#SBATCH --cpus-per-task=32    # number of OpenMP processes
+#SBATCH --mem=128G #TODO : to change
```

Comment:
@jcohenadad same thing for memory, is it for all results or only of a sub-dataset?

Reply:
sub-dataset